i18n_routable.gemspec

```diff
@@ -18,9 +18,10 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", '~> 3.2.0'
-  s.add_dependency 'i18n', '~> 0.6', '< 0.7'
+  s.add_dependency "rails", '~> 4.2.0'
+  s.add_dependency 'i18n', '~> 0.7'
```

Comment:
Where did you see that it's unsupported? I added it to I18n.available_locales and then set up this test which passed:
```
it "should accept 'en-US' :locale param as an option" do
      posts_path(:locale => :'en-US').should eql "/en-US/posts"
end
```